jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletChannel.java

```diff
@@ -539,13 +539,13 @@ public void handle()
 
                     case READ_CALLBACK:
                     {
-                        _context.run(() -> _servletContextRequest.getHttpInput().run());
+                        _context.run(_servletContextRequest.getHttpInput()::readCallback);
                         break;
                     }
 
                     case WRITE_CALLBACK:
                     {
-                        _context.run(() -> _servletContextRequest.getHttpOutput().run());
+                        _context.run(_servletContextRequest.getHttpOutput()::writeCallback);
```

Comment:
```suggestion
                        _context.run(_servletContextRequest.getHttpOutput());
```

Reply:
I prefer the former as it is more readable